packages/front-end/pages/fact-metrics/[fmid].tsx

```diff
@@ -645,7 +645,7 @@ export default function FactMetricPage() {
               open={() => setEditOpen(true)}
               canOpen={canEdit}
             >
-              {factMetric.windowSettings.delayValue > 0 && (
+              {factMetric.windowSettings.delayValue && (
```

Comment:
Don't do this.  `0 && ...` will be evaluated to `0` and React will print that to the screen.  If you force it to be a boolean (e.g. by doing `> 0`), then it will evaluate to `false` instead, which React will NOT print to the screen.  Alternatively, turn this into a ternary operator and do `delayValue ? (...) : null` to be explicit about what to output when it is zero.